docs/developer/service-quadlets.md

```
@@ -261,22 +261,11 @@ The RPM upgrade process includes:
-
-```bash
-# Simulate downgrade to validate which versions will be replaced
-sudo dnf downgrade flightctl-services --assumeno
-
-# Downgrade to the previous version
-sudo dnf downgrade flightctl-services
-
-# Or downgrade using a specific older RPM file
-sudo rpm -Uvh --oldpackage flightctl-services-<older-version>.rpm
-```
+> [!NOTE] 
```

markdownlint-cli2 (0.18.1)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)